appcues/src/main/java/com/appcues/ui/composables/CompositionRemembers.kt

```diff
@@ -65,13 +65,6 @@ internal fun LaunchOnShowAnimationCompleted(block: () -> Unit) {
     }
 }
 
-// we could make this public to give more flexibility in the future
-@Composable
-internal fun rememberAppcuesContentVisibility() = LocalExperienceCompositionState.current.let { remember { it.isContentVisible } }
-
-@Composable
-internal fun rememberAppcuesBackdropVisibility() = LocalExperienceCompositionState.current.let { remember { it.isBackdropVisible } }
```

Comment:
we discovered these remembers are not needed - only the reference to the MutableTransitionState within, for the animated visibility usage.